admin.Dockerfile

```diff
@@ -34,4 +34,4 @@ RUN chmod 777 /app/run_admin.sh
 # Don't run as root user
 USER 1000
 CMD /app/run_admin.sh
-# CMD [ "python3", "-m" , "flask", "run", "--host=0.0.0.0" "--port=5001"]
\ No newline at end of file
+# CMD [ "python3", "-m" , "flask", "run", "--host=0.0.0.0" "--port=5001"]
```

Comment:
🤔 ??

Reply:
sorry this was me accidentally committing admin.dockerfile and not knowing how to undo this file commit